package.json

```diff
@@ -7,7 +7,8 @@
     "lint:js": "eslint .",
     "lint:css": "stylelint blocks/**/*.css styles/*.css",
     "lint": "npm run lint:js && npm run lint:css",
-    "start": "aem up --url https://main--aem-boilerplate-commerce--hlxsites.aem.page",
+    "start": "aem up --url https://main--boilerplate-paas--adobe-commerce.aem.live",
+    "cypress:saas": "aem up --url https://main--boilerplate-accs--adobe-commerce.aem.live/",
```

Comment:
I don't think we need these - you don't even have to use `start` in cypress. I'll comment there to show you.